.changeset/hip-comics-agree.md

```diff
@@ -0,0 +1,15 @@
+---
+"@sumup-oss/circuit-ui": major
+---
+
+Marked the `ColorInput` and `PhoneNumberInput` components as stable. Update the related imports:
+
+```diff
+- import { ColorInput, ColorInputProps } from '@sumup-oss/circuit-ui/experimental';
++ import { ColorInput, type ColorInputProps } from '@sumup-oss/circuit-ui';
```

Comment:
I find it confusing to include the `type` annotation in the new import but not the old one
```suggestion
-import { ColorInput, type ColorInputProps } from '@sumup-oss/circuit-ui/experimental';
+import { ColorInput, type ColorInputProps } from '@sumup-oss/circuit-ui';
```